apps/web/screens/ApiCatalogue/ApiCatalogue.tsx

```diff
@@ -200,19 +204,96 @@ const ApiCatalogue: Screen<ApiCatalogueProps> = ({
     },
   ]
 
+  const navigationItems = [
+    {
+      active: true,
+      href: nn('linkServices'),
+      title: nn('linkServicesText'),
+      items: [
+        {
+          active: true,
+          title: nn('linkServiceListText'),
+        },
+        {
+          href: nn('linkDesignGuide'),
+          title: nn('linkDesignGuideText'),
+        },
+      ],
+    },
+    {
+      href: nn('linkIslandUI'),
+      title: nn('linkIslandUIText'),
+    },
+    {
+      href: nn('linkDesignSystem'),
+      title: nn('linkDesignSystemText'),
+    },
+    {
+      href: nn('linkContentPolicy'),
+      title: nn('linkContentPolicyText'),
+    },
+  ]
+
   return (
     <SubpageLayout
       main={
-        <SidebarLayout sidebarContent={<>Navigation goes here</>}>
+        <SidebarLayout
+          sidebarContent={
+            <Navigation
+              baseId="service-list-navigation"
+              colorScheme="blue"
+              items={navigationItems}
+              title={nn('linkThrounText')}
+              titleLink={{
+                active: true,
+                href: nn('linkThroun'),
+              }}
+            />
+          }
+        >
           <SubpageMainContent
             main={
               <Box>
-                <Box marginBottom={2}>
+                <Box display={['inline', 'inline', 'none']}>
+                  {/* Show when a device */}
+                  <Box paddingBottom="gutter">
+                    <Link href={nn('linkServices')}>
+                      <Button
```

Comment:
`<button>`s shouldn't be inside `<a>`s. A quick glance of other island.is applications shows that many people are using anchors inside buttons... this is something that has to be fixed.

I would either use `Link from 'next/link'` and do this:

```js
<Link href="/bla>
  <Button as="a" ...>...</Button>
</Link>
```

or use Link form island-ui and do a `router.push` in `onClick`

Reply:
Wrapping with <NextLink> and <a>